nolearn/lasagne/visualize.py

```diff
@@ -259,3 +260,110 @@ def saliency_map_net(net, X):
 
 def plot_saliency(net, X, figsize=(9, None)):
     return _plot_heat_map(net, X, figsize, lambda net, X, n: -saliency_map_net(net, X))
+    
```

Comment:
I'm surprised this doesn't give you a pep8 error, but what we're doing right now is we put two empty lines between functions.  Thanks.